shell/artifacts/Common/source/CopyCollection.js

```diff
@@ -0,0 +1,23 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+'use strict';
+
+defineParticle(({Particle}) => {
+  return class CopyCollection extends Particle {
+    setViews(views) {
+      this.on(views, 'input', 'change', e => {
+        let inputHandle = views.get('input');
+        inputHandle.toList().then(input => {
+          input.forEach(elem => views.get('output').store(elem));
+          this.relevance = Math.max(input.length, 5);
```

Comment:
Curious, why 5? Maybe worth a comment since I wondered. What is the expected range for relevance, 0..10?

Reply:
the relevance in particles is still kind of random. the main thing is for it to be non-zero (in either one of the particles in the recipe), otherwise it doesn't really matter. 
actually we already cap it on the outerPec side, so i removed the Math.max to keep the level of random to the minimum, and added a todo.